sktime/performance_metrics/annotation/metrics.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+"""
+Metrics for evaluating performance of segmentation estimators.
+
+Metrics are suitable for comparing predicted change point sets
+against true change points and quantify the error.
+"""
+
+import numpy as np
+import numpy.typing as npt
+from scipy.spatial.distance import directed_hausdorff
+from sklearn.utils import check_array
+
+__author__ = ["lmmentel"]
+
+
+def annotation_error(
+    true_change_points: npt.ArrayLike, pred_change_points: npt.ArrayLike
+) -> float:
+    """
+    Annotation error measuring discrepancy in the number of change points.
```

Comment:
```suggestion
    Annotation error counting the difference in the number of change points.
```
- 'discrepancy' to 'difference' is simpler language and more specific
- 'counting' is more specific than 'measuring'.